### DISCUSSION TITLE
출석체크 관련~!

### DISCUSSION THREAD

**Discussion by phphll** [fusiondevkr/pl900#22] [1 Q&A]
![image](https://user-images.githubusercontent.com/68364002/126159937-21764ed8-4379-47d7-8dfe-54b6b57ecac0.png)
퇴근은 따로없고 출근 저렇게 로그만 남아있으면되나용 ?

**Comment by justinyoo** [comment]
퇴... 퇴근까지 만들까효? ㄷㄷㄷ

일단은 출췍만 하는 걸로 하겠습니다!
